src/shared/api/daoService/daoService.ts

```diff
@@ -40,10 +41,15 @@ class DaoService extends AragonBackendService {
             return dao as IDao;
         }
 
-        const { network, address } = this.parseDaoId(dao.id);
-        const plugins = await pluginsService.getPluginsByDao({ urlParams: { network, address } });
+        try {
+            const { network, address } = this.parseDaoId(dao.id);
+            const plugins = await pluginsService.getPluginsByDao({ urlParams: { network, address } });
 
-        return { ...dao, plugins };
+            return { ...dao, plugins };
+        } catch (error) {
+            monitoringUtils.logError(error);
+            return { ...dao, plugins: [] };
+        }
```

Comment:
Not sure if this is backend improvement but we should probably be able to handle no plugins before the catch block as this is potentially not an error case, definitely an edge case however. Let's define the paths from Evan, I can't remember the specific reqs of this DAO impl.